File: Proj2/java/src/com/hw/PegBoardMove.java
package com.hw;

/**
* Representation of a move allowed in the game.
* The rowMove and colMove are based on the representation of the PegBoard array.
*
* @Author Joonho Kim
*/
public enum PegBoardMove {
  LEFT(0, -2),
  RIGHT(0, 2),
  TOP_LEFT(-2, -2),
  TOP_RIGHT(-2, 0),
  BOTTOM_LEFT(2, 0),
  BOTTOM_RIGHT(2, 2);
  
  private int rowMove, colMove;
  
  PegBoardMove(int rowMove, int colMove) {
    this.rowMove = rowMove;
    this.colMove = colMove;
  }
  
  public int getRowMove() {
    return rowMove;
  }
  
  public int getColMove() {
    return colMove;
  }
}

File: Proj1/python2/PegBoardMove.py

class PegBoardMove:
    rowMove = int()
    colMove = int()

    def __init__(self, rowMove, colMove):
        self.rowMove = rowMove
        self.colMove = colMove
    

PegBoardMove.LEFT = PegBoardMove(0, -2)
PegBoardMove.RIGHT = PegBoardMove(0, 2)
PegBoardMove.TOP_LEFT = PegBoardMove(-2, -2)
PegBoardMove.TOP_RIGHT = PegBoardMove(-2, 0)
PegBoardMove.BOTTOM_LEFT = PegBoardMove(2, 0)
PegBoardMove.BOTTOM_RIGHT = PegBoardMove(2, 2)
PegBoardMove.LIST = [
    PegBoardMove.LEFT, 
    PegBoardMove.RIGHT,
    PegBoardMove.TOP_LEFT,
    PegBoardMove.TOP_RIGHT,
    PegBoardMove.BOTTOM_LEFT,
    PegBoardMove.BOTTOM_RIGHT]